### DISCUSSION TITLE
Deploy on Vercel

### DISCUSSION THREAD

**Discussion by hotshinydev** [Finalgoal231/finalGoal#144] [Discuss about technical stack] [answered]
We are on deploying of this project.

**Comment by silvershiny** [comment, marked as answer]
This project is MERN stack.
Frontend: React.js - just deploy on Vercel
Backend: Node.js - add vercel.json and set environment variable and deploy on vercel
MongoDB: MongoDB - create cluster at mongo.com and link this with backend
